pages/scenarios/[runId].vue

```diff
@@ -13,8 +13,42 @@
           Rotate your mobile device to landscape for the best experience.
         </p>
       </CAlert>
-      <div v-show="appStore.largeScreen && appStore.currentScenario?.parameters && appStore.metadata?.parameters" class="card horizontal-card ms-auto">
+      <CModal
+        :visible="parameterModalVisible"
+        aria-labelledby="modalTitle"
+        @close="() => { parameterModalVisible = false }"
+      >
+        <CModalHeader>
+          <CModalTitle id="modalTitle">
+            Edit parameters
+          </CModalTitle>
+        </CModalHeader>
+        <CModalBody>
+          <ParameterForm :in-modal="true" />
+        </CModalBody>
+      </CModal>
+      <div v-show="appStore.currentScenario?.parameters && appStore.metadata?.parameters" class="card horizontal-card ms-auto parameters-card">
         <CRow>
+          <div
+            v-show="!appStore.largeScreen"
+            class="card-header h-100 align-content-center"
+            role="button"
+            @click="() => { parameterModalVisible = true }"
+          >
+            <CTooltip
+              content="Edit parameters"
+              placement="top"
+            >
+              <template #toggler="{ togglerId, on }">
+                <span :aria-describedby="togglerId" v-on="on">
+                  <CButton color="light">
+                    Parameters
+                  </CButton>
+                  <CIcon icon="cilPencil" class="form-icon link-secondary" />
+                </span>
+              </template>
+            </CTooltip>
+          </div>
```

Comment:
There's a bit of duplication here with the Parameters control markup underneath for large screen. Could pull out the functional bits  (including the modal) into an `EditParameters` component and include it in the right parent/with the right styling here depending on screen size. Also helps keep the sprawl in this page under control a bit.

Reply:
True

Reply:
Made the suggested component